src/fsharp/FSharp.Core/prim-types-prelude.fs

```diff
@@ -31,37 +31,86 @@ namespace Microsoft.FSharp.Core
     type uint = uint32
 
     type ``[]``<'T> = (# "!0[]" #)
+
     type ``[,]``<'T> = (# "!0[0 ...,0 ...]" #)
+
     type ``[,,]``<'T> = (# "!0[0 ...,0 ...,0 ...]" #)
+
     type ``[,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
     type ``[,,,,,,,,,,,,]``<'T> = (# "!0[0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...,0 ...]" #)
+
```

Comment:
You had to make so I got to see these lines again.  These all make me feel sad.

Reply:
:-)